src/kemCombiner_MLKEM1024_ECDH_P384_HMAC_SHA512.md

```diff
@@ -2,37 +2,40 @@ Example of id-MLKEM1024-ECDH-P384-HMAC-SHA512 Combiner function output.
 
 # Inputs
 mlkemSS:
-49c4f3099b31f36fc94fa2efe276d7fd7abedbb61e56c7a79e511674fa1ea534
+762ebf90869a30f797d490c54e5a86064edabeb1e765c0f5c05cb62dc78f64d8
 
-tradSS:  dda532717dba9a4d185854895e1e03019f11333365c8d9d2a80ee6a9ca572
-3c31b82c04e58c1ab140219b4ab8103c5de
+tradSS:  87f97859b24899bdebf1f243d68b3db37ad30b55a46d2bdaa4f690c28c
+c70a03f2943e98a58df1ab1056708a24c659b4
 
-tradCT:  040a51b44e9089e4e27901295eb3bad5834e1b3c3d123c9ced7483cd8620d
-ee344e54ae728f50f2012a55b9670abf55e3493b4b81842bf3a61602b576c58a6fe582
-45a6a4d03e646ee5ec9a2bfd2fb2c64ea7813b5303530ef7a297c438d19d62e
+tradCT:  044c0c543e6835f7661c1fb84ff7ffe35118981e7db0936847aac7d85d
+a72c1dc7a5fc1388083b7ea2ef1eede3a1dccea3638a59dc52f2fb2c883a55fdda8
+35c02789637216c633799e39e693e67b09f4cf6473d2dc0e4fdd3d8e8ef8de43d6f
+99
 
-tradPK:  04e2f1f9e5b13db94556e5384471eb586f8ac0d5e9bd8e49de859634a33d5
-6dc159325bda6506a00ae2407c5bfbad1d76ddf29cd70e8fb77d3ba01bdaed67d93624
-e5c62c9979203ea65564889df4f04a78d49878f62787d1047db3d3686acce09
+tradPK:  04c5b843ce6ccb59f9d419097d6f0791e758f414aab9569908cc9c7a48
+3cb62981186b7d33b180e1b70d8d9822018b1bda614d171bdb7ab6e0c2bdbe43dcc
+f9439dfa3cefcc46bfb232b50bdfaad0630b04a83831f13195d3308c0ffe842ae66
+49
 
 Label:  QSF-MLKEM1024-P384-HMACSHA512
 
 
 # Combined KDF Input:
 #  mlkemSS || tradSS || tradCT || tradPK || Label
 
-Combined KDF Input: 49c4f3099b31f36fc94fa2efe276d7fd7abedbb61e56c7a79e
-511674fa1ea534dda532717dba9a4d185854895e1e03019f11333365c8d9d2a80ee6a9
-ca5723c31b82c04e58c1ab140219b4ab8103c5de040a51b44e9089e4e27901295eb3ba
-d5834e1b3c3d123c9ced7483cd8620dee344e54ae728f50f2012a55b9670abf55e3493
-b4b81842bf3a61602b576c58a6fe58245a6a4d03e646ee5ec9a2bfd2fb2c64ea7813b5
-303530ef7a297c438d19d62e04e2f1f9e5b13db94556e5384471eb586f8ac0d5e9bd8e
-49de859634a33d56dc159325bda6506a00ae2407c5bfbad1d76ddf29cd70e8fb77d3ba
-01bdaed67d93624e5c62c9979203ea65564889df4f04a78d49878f62787d1047db3d36
-86acce09QSF-MLKEM1024-P384-HMACSHA512
+Combined KDF Input: 762ebf90869a30f797d490c54e5a86064edabeb1e765c0f
+5c05cb62dc78f64d887f97859b24899bdebf1f243d68b3db37ad30b55a46d2bdaa4
+f690c28cc70a03f2943e98a58df1ab1056708a24c659b4044c0c543e6835f7661c1
+fb84ff7ffe35118981e7db0936847aac7d85da72c1dc7a5fc1388083b7ea2ef1eed
+e3a1dccea3638a59dc52f2fb2c883a55fdda835c02789637216c633799e39e693e6
+7b09f4cf6473d2dc0e4fdd3d8e8ef8de43d6f9904c5b843ce6ccb59f9d419097d6f
+0791e758f414aab9569908cc9c7a483cb62981186b7d33b180e1b70d8d9822018b1
+bda614d171bdb7ab6e0c2bdbe43dccf9439dfa3cefcc46bfb232b50bdfaad0630b0
+4a83831f13195d3308c0ffe842ae6649QSF-MLKEM1024-P384-HMACSHA512
```

Comment:
Similar to the X-wing label, the "QSF-MLKEM1024-P384-HMACSHA512" portion of this should be in hexadecimal (presumably there's a way to fix this globally, just commenting here so it's easy to verify later)